src/styles/fonts/_roboto.scss

```diff
@@ -0,0 +1,26 @@
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Regular.woff2) format('woff2');
+  src: url(../fonts/Roboto-Regular.ttf) format('truetype');
+  font-weight: 400;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Medium.woff2) format('woff2');
+  src: url(../fonts/Roboto-Medium.ttf) format('truetype');
+  font-weight: 500;
+  font-style: normal;
+  font-display: swap;
+}
+
+@font-face {
+  font-family: Roboto;
+  src: url(../fonts/Roboto-Bold.woff2) format('woff2');
+  src: url(../fonts/Roboto-Bold.ttf) format('truetype');
```

Comment:
Again, for the Roboto-Bold font, combine the `woff2` and `ttf` formats into a single `src` declaration.